app/social/components/sexualPartnerForm.html

```diff
@@ -0,0 +1,44 @@
+<form class="row" name="form" ng-submit="vm.submit(form);" novalidate>
```

Comment:
Better if we show consistency of design:
![image](https://user-images.githubusercontent.com/3810265/38797019-8724f6e8-415d-11e8-947f-dfdda7d03384.png)
and
![image](https://user-images.githubusercontent.com/3810265/38797245-302149c2-415e-11e8-9428-caeaaef1711b.png)
 
With one save button in the end (Next = Save).